### DISCUSSION TITLE
关于英语输入的问题

### DISCUSSION THREAD

**Discussion by keaidu** [imfuxiao/Hamster#289] [Ideas]
现在输入法已经有了英语输入模式了，只是不能联想输入而已，可我看见雾凇输入法自带了英语输入方案，能不能将雾凇输入法的英语输入方案直接内置到英语输入模式里面？

**Comment by imfuxiao** [comment]
英文输入需要英文词库，雾凇输入方案内置了英文词库。

我的意思是，在雾凇输入方案下，中英混合输入可以满足输入需求。就没有必要在英文模式下内置词库了。
